instance/factories.py

```diff
@@ -151,6 +151,7 @@ def production_instance_factory(**kwargs):
         # Allow production instances to use a different OpenStack instance flavor by default.
         # This allows using a larger instance flavor for production instances.
         openstack_server_flavor=settings.OPENSTACK_PRODUCTION_INSTANCE_FLAVOR,
+        provision_failed_emails=settings.PROD_APPSERVER_FAIL_EMAILS,
```

Comment:
@swalladge, it will be a good idea to update the test(s) in `instance/tests/test_factories.py` to check this.